physical/postgresql/postgresql.go

```diff
@@ -120,6 +120,7 @@ func NewPostgreSQLBackend(conf map[string]string, logger log.Logger) (physical.B
 		return nil, errwrap.Wrapf("failed to connect to postgres: {{err}}", err)
 	}
 	db.SetMaxOpenConns(maxParInt)
+	db.SetMaxIdleConns(maxParInt)
```

Comment:
I don't think these should use the same config parameter because then this will force keeping _all_ connections open even when not needed. Can you add a max_idle_connections configuration parameter instead?

Reply:
> this will force keeping all connections open even when not needed.

This is actually what I'm hoping to accomplish, but I agree it doesn't seem right to push this as the default config.

> Can you add a max_idle_connections configuration parameter instead?

Absolutely. Should I make `max_open_connections` an alias param for `max_parallel` as well to have better config parity with the logical postgres database adapter?

Reply:
Not for now...at the storage layer it's all max_parallel so let's just keep that way for now.